src/scopes/validator.rs

```diff
@@ -68,6 +107,99 @@ impl Validator {
         }
     }
 
+    /// Returns a validator only containing the unmatched scopes.
```

Comment:
```suggestion
    /// Returns a validator only containing the unmatched scopes. If all scopes are matched, [`None`] is returned.
```

Not sure if an example for this is necessary. Maybe
```rust
let scopes = &[Scope::ChatEdit, Scope::ChatRead];
assert!(validator.missing(scopes).is_none());
``` 

in the first example.